BankManagment/UnitOfWork/UnitOfWork.cs

```diff
@@ -0,0 +1,47 @@
+﻿using BankManagmentDB.Context;
+using BankManagmentDB.Entity;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace BankManagmentDB
+{
+    public class UnitOfWork : IUnitOfWork
+    {
+        private readonly ApplicationDbContext _context;
+
+        public UnitOfWork(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        private IRepository<AccountType> _accountTypesRepository;
+        public IRepository<AccountType> AccountTypesRepository =>
+            _accountTypesRepository ??= new Repository<AccountType>(_context);
+
+        private IRepository<PaymentMethod> _paymentMethodsRepository;
+        public IRepository<PaymentMethod> PaymentMethodsRepository =>
+            _paymentMethodsRepository ??= new Repository<PaymentMethod>(_context);
```

Comment:
Generally in a WebAPI environment, services are registered with Scoped Lifetime, If you register IUnitOfWork as Scoped, then it would eliminate the need to check _paymentMethodsRepository for null. and as mentioned in previous comment, use DI